graders/python/python_autograder/code_feedback.py

```diff
@@ -117,6 +127,24 @@ def finish(cls, fb_text: str) -> NoReturn:
         cls.add_feedback(fb_text)
         raise GradingComplete("Your answer is correct.")
```

Comment:
👀 This seems weird? Can we remove this argument entirely to avoid confusion?

Reply:
I don't even think the exception message even gets read? Odd

Reply:
Yeah, I don't think it is, which should make it safe to just remove it. I'll do so.

Reply:
Removed in e975af4.